force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js

```diff
@@ -1,81 +1,90 @@
-import { api, LightningElement, track, wire } from 'lwc';
-import { getObjectInfo } from 'lightning/uiObjectInfoApi';
-import DATA_IMPORT from '@salesforce/schema/DataImport__c';
-import PAYMENT_FIELD from '@salesforce/schema/DataImport__c.Payment_Method__c';
-import donationHistoryDatatableAriaLabel from '@salesforce/label/c.donationHistoryDatatableAriaLabel';
-import RD2_ScheduleVisualizerColumnDate from '@salesforce/label/c.RD2_ScheduleVisualizerColumnDate';
-import getDonationHistory from '@salesforce/apex/DonationHistoryController.getDonationHistory';
-import commonAmount from '@salesforce/label/c.commonAmount';
-import donationHistoryDonorLabel from '@salesforce/label/c.donationHistoryDonorLabel';
+import { api, LightningElement, track } from "lwc";
+import donationHistoryDatatableAriaLabel from "@salesforce/label/c.donationHistoryDatatableAriaLabel";
+import RD2_ScheduleVisualizerColumnDate from "@salesforce/label/c.RD2_ScheduleVisualizerColumnDate";
+import getDonationHistory from "@salesforce/apex/DonationHistoryController.getDonationHistory";
+import commonAmount from "@salesforce/label/c.commonAmount";
+import donationHistoryDonorLabel from "@salesforce/label/c.donationHistoryDonorLabel";
 const RECORDS_TO_LOAD = 50;
 export default class DonationHistoryTable extends LightningElement {
+  
     @api contactId;
 
-    
-    paymentMethodLabel;
+  paymentMethodLabel;
 
-    totalNumberOfRows;
```

Comment:
There's a few indentation changes in this file that don't seem totally necessary, usually we've gone with 4 spaces for indentation. Can you reformat this file so we're always using 4 spaces for indentation?